code/game/dna/genes/powers.dm

```diff
@@ -100,6 +100,29 @@
 	..()
 	block = GLOB.shockimmunityblock
 
+/datum/dna/gene/basic/smallvox //This should not be acquirable in-game by players via Genetics as there is no assigned Structrural Enyzme for SmallVox
+	name= "SmallVox"
+	activation_messages = list("Now is smalls...")
+	deactivation_messages = list("Voxxy no longer feeling so small...Voxxy...talls?!")
+	instability = GENE_INSTABILITY_MINOR
+	mutation = SMALLVOX
+
+/datum/dna/gene/basic/smallvox/activate(mob/M, connected, flags)
+	..()
+	M.resize = 0.8
+	M.update_transform()
+	M_move_resist=M_FORCE_WEAK
+	
+/datum/dna/gene/basic/smallvox/deactivate(mob/M, connected, flags)
+	..()
+	M.resize = 1.25
+	M.update_transform()
+	M_move_resist = M_FORCE_NORMAL
```

Comment:
I am guessing you want M.move_resist

Reply:
> Also, having tested this, Vox are still gaining the normal dwarf gene from... somewhere else, because roundstart Vox have their `move_force` and `move_resist` both at 500.

Ugh, yeah. I might have missed one file. Let me delete this PR and resubmit

Reply:
@SteelSlayer  I had it all pinned down. I _thought_ I was missing a file in the PR!